lib/sql_footprint/sql_statements.rb

```diff
@@ -3,12 +3,17 @@
 
 module SqlFootprint
   class SqlStatements
+    NEWLINE = "\n".freeze
     def initialize
       @statements = Set.new
     end
 
     delegate :to_a, :add, :sort, :count, to: :statements
 
+    def to_s
+      statements.sort.join(NEWLINE) + NEWLINE
```

Comment:
@brandonjoyce Even better! Maybe `SqlStatementsSerializer` will have a `to_file` method on it?